psalm.xml

```diff
@@ -13,6 +13,10 @@
         <ignoreFiles>
             <directory name="test/TestAsset"/>
             <directory name="vendor"/>
+            <!-- can be removed once PHP8.0 support dropped -->
+            <file name="src/Strategy/BackedEnumStrategy.php"/>
```

Comment:
Instead of excluding, can we not expand the baseline? :thinking:

Reply:
Yeah. I've still got to exclude the test assets though, else we get parse errors.

Reply:
Ye, understandable, just thinking that at least for `src/`, we should probably keep scanning everything

Reply:
Can this particular file be kept in here, please? :D